src/stageCache.js

```diff
@@ -68,11 +68,14 @@ const createPatchForEndpoint = (endpointSettings, serverless) => {
   let lambda = serverless.service.getFunction(endpointSettings.functionName);
   if (isEmpty(lambda.events)) {
     serverless.cli.log(`[serverless-api-gateway-caching] Lambda ${endpointSettings.functionName} has not defined events.`);
+    return;
   }
-  // TODO there can be many http events
-  let httpEvents = lambda.events.filter(e => e.http != null);
+  let httpEvents = lambda.events.filter(e => e.http != undefined)
```

Comment:
I'm thinking this could be wrapped in a method named, for instance, `httpEventsOf(lambda)`, to keep the code more readable.

Reply:
You're right. I have updated the file.